### PR TITLE
Feat/sudoku validation

### PR DESCRIPTION
**Pull request by patrikglanemann**
Just added a simple submit button.
Now you can validate the Sudoku and for now see the status of it. (This will be changed in the future)

What I did:
- Add simple button
- Add logic to fetch (POST) status of validation
(The author of the API I am using documented an example on how to do that, since you have to use application/x-www-form-urlencoded as the content type. I simply copied it and changed the variables.)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -5,6 +5,5 @@
 .Content {
   display: flex;
   flex-direction: column;
-  text-align: center;
-  justify-content: center;
+  align-items: center;
 }
```
Comment (hellodebs) [inline src/App.css:9]: maybe you are going to do it anyway but perhaps create a margin-top? :)

**Review by hellodebs — COMMENTED** [review]
Looks really good, Patrik! :D Good process!

File: src/pages/SudokuPage.js [hunk 2]
```diff
@@ -67,6 +69,35 @@ export default function SudokuPage() {
...
 
+  function handleSubmitClick() {
+    const encodeBoard = (board) =>
+      board.reduce(
+        (result, row, i) =>
+          result +
+          `%5B${encodeURIComponent(row)}%5D${
+            i === board.length - 1 ? "" : "%2C"
+          }`,
+        ""
```
[the remark this answers is not in the document]
Comment (patrikglanemann) [reply src/pages/SudokuPage.js:80]: Well, I didn't looked much into it. I just copied what the author of the API posted as an example to make it work with content type application/x-www-form-urlencoded 😅

File: src/pages/SudokuPage.js [hunk 3]
```diff
@@ -67,6 +69,35 @@ export default function SudokuPage() {
...
+    };
+
+    fetch("https://sugoku.herokuapp.com/validate", {
+      method: "POST",
+      body: encodeParams(data),
+      headers: { "Content-Type": "application/x-www-form-urlencoded" },
+    })
+      .then((response) => response.json())
+      .then((response) => setSudokuStatus(response.status))
+      .catch(console.warn);
```
[the remark this answers is not in the document]
Comment (patrikglanemann) [reply src/pages/SudokuPage.js:98]: I think the only difference is that .error gets shown under errors and .warn under warnings in the console:)

File: src/App.css (hunk 1 above)
Comment (patrikglanemann) [reply src/App.css:9]: Yes you are right. I only put in the bare minimum for styling and will put some attention to that stuff later on:)

**Pull request merged**